lib/templates_helpers/at_nav_button.js

```diff
@@ -10,6 +10,7 @@ AT.prototype.atNavButtonEvents = {
         if (Meteor.user())
             AccountsTemplates.logout();
         else
+            event.preventDefault();
```

Comment:
That's a good point, sorry I was being a little hurried with it. I'll update the PR.